### PR TITLE
Update language_hu.h

### PR DESCRIPTION
**Pull request by AntoszHUN**
Fixed language errors in the Hungarian language pack.

Made By: AntoszHUN

### CODE REVIEW

File: Marlin/src/lcd/language/language_hu.h [hunk 1]
```diff
@@ -318,7 +318,7 @@ namespace Language_hu {
   PROGMEM Language_Str MSG_TEMPERATURE                     = _UxGT("Höfok");
   PROGMEM Language_Str MSG_MOTION                          = _UxGT("Mozgatások");
   PROGMEM Language_Str MSG_FILAMENT                        = _UxGT("Nyomtatószál");
-  PROGMEM Language_Str MSG_VOLUMETRIC_ENABLED              = _UxGT("E mm") SUPERSCRIPT_THREE _UxGT("-ben");
```
Comment (rhapsodyv) [inline Marlin/src/lcd/language/language_hu.h:321]: There's a reason for a separated `SUPERSCRIPT_THREE`: font support. You should keep it.

File: Marlin/src/lcd/language/language_hu.h [hunk 2]
```diff
@@ -56,7 +56,7 @@ namespace Language_hu {
   PROGMEM Language_Str MSG_MAIN                            = _UxGT("<Fömenü>");
   PROGMEM Language_Str MSG_ADVANCED_SETTINGS               = _UxGT("További Beállítások");
   PROGMEM Language_Str MSG_CONFIGURATION                   = _UxGT("Konfiguráció");
-  PROGMEM Language_Str MSG_RUN_AUTO_FILES                  = _UxGT("Autoinditás");
```
Comment (rhapsodyv) [inline Marlin/src/lcd/language/language_hu.h:59]: There's no `MSG_AUTOSTART` any more. It got renamed to `MSG_RUN_AUTO_FILES`

File: Marlin/src/lcd/language/language_hu.h [hunk 3]
```diff
@@ -627,4 +627,7 @@ namespace Language_hu {
 #else
   #define MSG_FIRST_FAN_SPEED       MSG_FAN_SPEED_N
   #define MSG_EXTRA_FIRST_FAN_SPEED MSG_EXTRA_FAN_SPEED_N
+#endif
+  #define MSG_FIRST_FAN_SPEED       MSG_FAN_SPEED_N
```
Comment (rhapsodyv) [inline Marlin/src/lcd/language/language_hu.h:631]: This is repeated. Should be removed.

**Comment by ghost** [comment]
I understand. Then when I have time I'll fix it.

**Comment by thinkyhead** [comment]
Thanks! I was able to merge the good changes. There have been many new items added if you ever feel like doing more.

**Comment by ghost** [comment]
Thanks! Over the weekend, I will review and repair if necessary.

> Thanks! I was able to merge the good changes. There have been many new items added if you ever feel like doing more.

Thanks! Over the weekend, I will review and repair if necessary.

**Pull request closed without merging**